### ISSUE TITLE
Section 508 Accessibility Audit

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12095]
This epic tracks the effort to correct issues surfaced by the [Section 508 Accessibility Audit on the Caseflow Demo website](https://dsva.slack.com/files/U51RH55BR/FMSE83EH1/appeals_modernization_bva_caseflow_quick_test_20190904-lkd-1_final.docx
). Most issues are related to inadequate support for using the website with assistive technology (AT), the most common of which are [screen readers](https://accessibility.digital.gov/front-end/screen-reader/) and [keyboard access](https://accessibility.digital.gov/front-end/keyboard-access/).

To reproduce the issues:
1. Enable a screen reader on your computer.
Mac: https://www.imore.com/how-enable-voiceover-mac
Windows: https://ccm.net/faq/43342-how-to-enable-the-screen-reader-in-windows-10
2. Load the product/page on Chrome where the issue is located.
3. Use the tab key to move through elements on the webpage and find the affected UI element. Check the specific issue for expected behavior (i.e. what should the screen reader be saying vs. what it actually is saying).

Support required:
- Browser: only Chrome

Full Report: https://dsva.slack.com/files/U51RH55BR/FMSE83EH1/appeals_modernization_bva_caseflow_quick_test_20190904-lkd-1_final.docx

Issues: https://airtable.com/tbl0x6Zrjg7Nn9dJc/viwiuOHVUrUQ4T1dP?blocks=hide

**Comment by alisan16** [comment]
Closing this ticket as the audit from 2019 has been completed 🎉 
That being said, Caseflow has undergone another Section 508 audit and will be working through those issues shortly. Some context here: https://dsva.slack.com/archives/C7P7M26QM/p1610128636309500
